dashboard/test/ui/features/step_definitions/ailab_steps.rb

```diff
@@ -0,0 +1,13 @@
+When /^I select table column "([^"]*)"$/ do |column|
```

Comment:
I know this is in an AI Lab specific set of steps, but the name does feel a bit more generic than the functionality.  Maybe we can refer to it as "data table" rather than just a "table"?